factcast-store/src/main/java/org/factcast/store/registry/transformation/cache/PgTransformationCache.java

```diff
@@ -176,6 +184,19 @@ public void compact(@NonNull ZonedDateTime thresholdDate) {
                 new Date(thresholdDate.toInstant().toEpochMilli())));
   }
 
+  @Override
+  public void invalidateTransformationFor(String ns, String type) {
+    // we clear the buffer and flush only accesses
+    // we don't flush new transformations in the cache as those could contain the invalidated one
+    // could be optimized
+    clearAndFlushAccessesOnly();
+
+    jdbcTemplate.update(
+        "DELETE FROM transformationcache WHERE header ->> 'ns' = ? AND header ->> 'type' = ?",
+        ns,
+        type);
+  }
+
```

Comment:
create transaction, and after commit flush the buffer again

Reply:
consider blacklisting for a x amount of time to avoid in-flight writes to be persisted